### ISSUE TITLE
"Windows bat relies on miniconda which can not be silently installed under a path with spaces"

### ISSUE THREAD

**Issue by DuckersMcQuack** [oobabooga/text-generation-webui#1468]
### Describe the bug

"This script relies on Miniconda which can not be silently installed under a path with spaces.
Press any key to continue . . .
"

The error i'm getting. Being pretty "new" to python and anaconda, so this one is unknown to me how to fix.

Using windows 11.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Doubleclick to run. Pretty much all i did.

### Screenshot

![WindowsTerminal_PlzwZeuLFC](https://user-images.githubusercontent.com/9039370/233754647-9eabb00c-31f7-4f9e-89e1-0b51c89e6858.png)

### Logs

```shell
Nothing made from what can't be run.
```

### System Info

```shell
Windows 11 home
Installed on SN850 1TB
5900x
64GB ram
RTX 3090 suprim X
```

**Comment by wintergrey** [comment]
As stated in the error message, there cannot be a space in the names of any of the folders that the program is in.
"language model" has a space in its name.
Either rename that folder to something like "language_model"
or (highly recommended) move your "oobabooga_windows" folder directly onto the c:\ drive
ie. Open "This PC". Open "Windoes Elven (C:)". Drag your existing "oobabooga_windows" folder into that window.
After that the install will work correctly & you'll avoid having to deal with windows path length issues you'd likely be encountering later on anyway.

**Comment by Renegadesoffun** [comment]
i had the same issue  thanks!

**Comment by SenrirenTokyo** [comment]
That worked! Thank you @wintergrey!

**Comment by edd-ie** [comment]
Thanks, removing the spaces helped.

**Comment by CrazyFanatic** [comment]
> As stated in the error message, there cannot be a space in the names of any of the folders that the program is in. "language model" has a space in its name. Either rename that folder to something like "language_model" or (highly recommended) move your "oobabooga_windows" folder directly onto the c:\ drive ie. Open "This PC". Open "Windoes Elven (C:)". Drag your existing "oobabooga_windows" folder into that window. After that the install will work correctly & you'll avoid having to deal with windows path length issues you'd likely be encountering later on anyway.

I've tried this but it says miniconda is unable to download or run on my PC
Is there a solution for my issue?

**Comment by NiTiResearcher** [comment]
Thanks, it's worked!

**Comment by Nevadead** [comment]
Hey, all!
I'm encountering the issue but I have changed all the spaces to "_" 

![image](https://github.com/oobabooga/text-generation-webui/assets/38713190/b985653d-a21f-4c33-a22b-96feec369b61)

I've Tried all the drives too.

**Comment by lvillalta100** [comment]
> As stated in the error message, there cannot be a space in the names of any of the folders that the program is in. "language model" has a space in its name. Either rename that folder to something like "language_model" or (highly recommended) move your "oobabooga_windows" folder directly onto the c:\ drive ie. Open "This PC". Open "Windoes Elven (C:)". Drag your existing "oobabooga_windows" folder into that window. After that the install will work correctly & you'll avoid having to deal with windows path length issues you'd likely be encountering later on anyway.

This was the solution for me. Thank a lot

**Comment by 2-fly-4-ai** [comment]
Its still relevant and will always be relevant until there is some easy way to bypass this. I shouldn't have to rename my windows users to run a program.

**Comment by Lakshagrawal** [comment]
Thanks That work

**Comment by borioli** [comment]
This script relies on Miniconda which can not be silently installed under a path with spaces.
Press any key to continue. .

Guys! I can't find the directory. I've done everything... I can't find anything about the programme! I don't know what else to do... Anyone who can help me, I'd appreciate it...

**Comment by PandaBTBs** [comment]
> This script relies on Miniconda which can not be silently installed under a path with spaces. Press any key to continue. .
> 
> Guys! I can't find the directory. I've done everything... I can't find anything about the programme! I don't know what else to do... Anyone who can help me, I'd appreciate it...

Are you sure that text-generation-webui is located in a folder that does not have spaces in its name?

**Comment by richardstevenhack** [comment]
> Its still relevant and will always be relevant until there is some easy way to bypass this. I shouldn't have to rename my windows users to run a program.

Absolutely! This is ridiculous. Requiring the end user to change where he wants the program to run and rename folders he is using for specific reasons is simply bad end user design.

Dump miniconda and change the install to use pipx. Or separate out the miniconda install.

**Comment by CoolAshraf** [comment]
dont do any thing crazy - just copy your directory on the desktop and run your setup from there " no more minoconda error now " enjoy

**Comment by uauu437** [comment]
> As stated in the error message, there cannot be a space in the names of any of the folders that the program is in. "language model" has a space in its name. Either rename that folder to something like "language_model" or (highly recommended) move your "oobabooga_windows" folder directly onto the c:\ drive ie. Open "This PC". Open "Windoes Elven (C:)". Drag your existing "oobabooga_windows" folder into that window. After that the install will work correctly & you'll avoid having to deal with windows path length issues you'd likely be encountering later on anyway.

it actually worked for me

**Comment by Daxxel06** [comment]
Do anyone knows how to solve this error.
FileNotFoundError: [WinError 2] El sistema no puede encontrar el archivo especificado: 'roop-unleashed'

**Comment by PawAlldeller** [comment]
Terribly inconvenient error, I have the same one
